nginx.conf

```diff
@@ -266,6 +265,10 @@ http {
                 next-dev-oulu.digitransit.fi
                 next-dev-turku.digitransit.fi
                 next-dev-tampere.digitransit.fi
+		dev-tampere.digitransit.fi
+		tampere.digitransit.fi
+		repa.tampere.fi
+		reittiopas.tampere.fi
```

Comment:
I would prefer these to be on the same line with the next-dev-tampere domain so it's more easy to spot domains that point which all domains are used for the same city.